plugins/mcreator-localization/help/default/entity/step_height.md

```diff
@@ -0,0 +1,2 @@
+Sets the step height of the mob.

```

Comment:
We try to uniformize all words, so `mob` should be changed for `entity` (or `living entity`).
```suggestion
Sets the step height of the entity.

```